### PR TITLE
Revert "feat: add autoscaler_policy_overrides support to module (#92)"

### PR DESCRIPTION
**Pull request by vladklokun**
This reverts commit 1bc00679bad9f436ed2fc6c757cefba4f49fd57c.

Requires castai/terraform-provider-castai#328.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -464,9 +335,9 @@ terraform-docs markdown table . --output-file README.md
 
 | Name | Version |
 |------|---------|
-| <a name="provider_castai"></a> [castai](#provider\_castai) | 7.3.0 |
-| <a name="provider_helm"></a> [helm](#provider\_helm) | 2.13.2 |
-| <a name="provider_null"></a> [null](#provider\_null) | 3.2.2 |
+| <a name="provider_castai"></a> [castai](#provider\_castai) | ~> 7.3.0 |
+| <a name="provider_helm"></a> [helm](#provider\_helm) | >= 2.0.0 |
+| <a name="provider_null"></a> [null](#provider\_null) | n/a |
```
Comment (vladklokun) [inline README.md:340]: question: these version values seem to differ from the changes introduced in #93 

I generated these automatically with `make generate-doc`. Is there a reason #93 produced different values and that tthe `make generate-doc` produces different values after this revert??

cc @aldor007

Comment (vladklokun) [reply README.md:340]: I would appreciate your take too @linkas45 :pray:

Comment (linkas45) [reply README.md:340]: I think it was not generated, but changed manually. I would say it's good to go as this release is still blocking customers

Comment (vladklokun) [reply README.md:340]: Okay then, I’ll proceed.

**Pull request merged**